### PR TITLE
Help people who try to use an existing slug


### PR DESCRIPTION
**Pull request by gabebw**
## Clarify the instructions for joining
* Emphasize that they should not pick an existing slug
* Add angle brackets inside the URL so that if they try to copy-paste, it does not work.

CleanShot 2022-10-09 at 09 32 18@2x

## Show people a special page when they pick an existing slug

CleanShot 2022-10-09 at 09 30 39@2x

### CODE REVIEW

File: app/controllers/redirections_controller.rb [hunk 1]
```diff
@@ -100,7 +101,7 @@ def ensure_request_is_not_from_a_bot
   def ensure_request_is_not_a_subdomain_of_an_existing_site
     if referrer
       referrer_without_leading_subdomain = URI.parse(referrer).host.
-        sub(/^[^.]+\./, '')
+        sub(/^[^.]+\./, "")
```
Comment (edwardloveall) [inline app/controllers/redirections_controller.rb:104]: nice.

**Review by edwardloveall — APPROVED** [review]

File: app/controllers/redirections_controller.rb (hunk 1 above)
Comment (gabebw) [reply app/controllers/redirections_controller.rb:104]: All credit to rufo 😄 https://github.com/ruby-formatter/rufo

**Comment by edwardloveall** [comment]
We should definitely add this, but it may not fully solve the problem. Some people fill out one slug correctly and one incorrectly. They only need to click the correct one to join the 'ring.

**Comment by gabebw** [comment]
But when someone clicks the bad one (perhaps the original owner), they'll see this message. And then hopefully they'll email us. Or their friend will click it, and tell the owner.

**Comment by gabebw** [comment]
I think it's fine if they join with a good slug but have one broken link.

**Pull request merged**